packages/core/integration-test/integration-test.js

```diff
@@ -55,8 +58,9 @@ const runLocally = (event) => {
     handler(event, {}, (err, data) => {
       if (err) {
         reject(err);
+      } else {
+        resolve(data);
```

Comment:
Fixed a problem while I was here. We should call either `resolve` or `reject` but not both.